packages/components/src/image/image.tsx

```diff
@@ -60,10 +60,10 @@ const Base = styled(RNImage, {
     radius: {
       none: {},
       12: {
-        borderRadius: 12,
+        borderRadius: '$12',
```

Comment:
Typescript says here that: "Type 'string' is not assignable to type 'number'". How do we want to bypass this?
It goes for both values in this file.